### DISCUSSION TITLE
German Language Leader

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#347] [Language Leaders]
@Scorpoon, @ludeeus, @mib1185, @osworx, @N3rdix, @TheDK, @easterapps, @Skorfulose - moved discussion here.

**Comment by osworx** [comment]
I am curious .. how is the current status?

**Reply by andreasbrett** [reply]
@Scorpoon seems to have been added as official language leader for German:
https://github.com/home-assistant/intents/blob/20a8a825ab6c6c3626c8cda2c09b2d9f4e855908/languages.yaml#L18

**Reply by TheDK** [reply]
@Scorpoon just let us know if you want to share some of the workload…

**Reply by Scorpoon** [reply]
Sure, i dont have to do it alone. I was on vacation, i hope i will get some work done today 👍

**Reply by TheDK** [reply]
Just to be sure: For contributing one forks the YAML file, edits it and creates a pull request as per usual? Any pointers you want to give as to focus areas / domains?

**Reply by mib1185** [reply]
Here you will finde a short guide about how to contribute:
https://github.com/home-assistant/intents/blob/main/CONTRIBUTING.md

**Reply by TheDK** [reply]
Yes, I read that, this is why I wrote "Just to be sure" - so I will edit directly via GitHub...

**Reply by mib1185** [reply]
keep in mind, when adding new sentences, you have to add tests, too 😉

**Comment by synesthesiam** [comment]
@TheDK @mib1185 @andreasbrett - anyone else interested in being a language leader for German? We have a few PR's that need attention.

**Reply by easterapps** [reply]
Yes, you can add me as a leader

**Reply by mib1185** [reply]
- yeah sure, please add meas well. -> #1088

**Comment by holger-tangermann** [comment]
Hallo, 
not sure if this is the right spot in the forum...
I just tried the 2 intents ... 
"öffne alle Abdeckungen"
"öffne alle Rolläden"
.. in an attempt to open all my covers. Assist responded "geöffnet" (see below) and opened indeed the covers. But it also switched on all lights, all electrical switches and even the siren.  

If I try to open a single cover entity (e.g. "Öffne Büro"), Assist response with "Entschuldigung, das habe ich nicht verstanden". 

However if I try to switch a single cover entity "schalte Büro an | aus", the cover is opened | closed indeed. 

Please let me know, if I can be of any help to further  improve the Assist funktion. 

Holger

![Screenshot_2023-02-02-14-01-05-11_c3a231c25ed346e59462e84656a70e50](https://user-images.githubusercontent.com/76705846/216333022-74e5a45c-29b0-4817-8f06-2b8a3b8712c3.jpg)

**Reply by mib1185** [reply]
Please report this as an [issue to the Home Assistant Core project](https://github.com/home-assistant/core/issues/new?assignees=&labels=&template=bug_report.yml)

**Comment by synesthesiam** [comment]
@easterapps @mib1185 when you get a chance, can you please look into translating the sentences for `HassGetState`?

**Reply by mib1185** [reply]
will have a look today evening (CET)

**Reply by synesthesiam** [reply]
Thanks @mib1185, I added you as a language leader.

**Reply by mib1185** [reply]
- ~I've started with it~ ready to review -> #1096
- but encounter an issue where the German `ß` is "translated" into `ss` in the response, but German umlauts (_ä ö ü_) were not translated :thinking: ... more details are noted down in the PR

**Reply by synesthesiam** [reply]
OK, I will take a look.

**Reply by mib1185** [reply]
> OK, I will take a look.

i've created an issue for that #1133

**Comment by synesthesiam** [comment]
@Scorpoon Since you haven't been active for a while, I'm going to remove you as a language leader. If you have time again in the future, let me know.
